R/trace_tests.R

```diff
@@ -190,7 +190,10 @@ update_current_test <- function() {
 
   has_srcfile <- viapply(syscall_srcfile, length) > 0L
   srcfile_tmp <- logical(length(has_srcfile))
-  srcfile_tmp[has_srcfile] <- startsWith(syscall_srcfile[has_srcfile], normalizePath(.libPaths()[[1]]))
+  srcfile_tmp[has_srcfile] <- startsWith(
+    syscall_srcfile[has_srcfile],
+    normalizePath(.libPaths()[[1]], mustWork = FALSE)
+  )
```

Comment:
Another sort of tangential bugfix. I found a situation where `normalizePath` would throw a "file or directory not found" error. I haven't discovered exactly what testing scenario somehow led to an invalid `.libPaths()`, but this is a simple hotfix just to safeguard against covr introducing errors into the test execution.